src/shipit_workflow/requirements.txt

```diff
@@ -0,0 +1,7 @@
+-e ./../../lib/cli_common[sentry] #egg=mozilla-cli-common
+-e ./../../lib/backend_common[log,security,cors,api,auth,db,cache,pulse] #egg=mozilla-backend-common
```

Comment:
i think you don't really need ``pulse`` feature, right?

Reply:
also ``cache`` feature is going to be used? I personally start with the smallest set of features and add them when needed.

Reply:
but you might probably add ``auth0`` feature.

Reply:
I tried to keep the requirements minimal, but couldn't generate the corresponding nix files because of taskcluster>2.0 which was pulled by default. I can try to remove those and regenerate the nix files.